proposals/nnnn-remote_atf_testing.md

```diff
@@ -0,0 +1,83 @@
+# Remote Automated testing
+
+* Proposal: [SDL-NNNN](NNNN-remote_atf_testing.md)
+* Author: [Alexander Kutsan](https://github.com/LuxoftAKutsan)
+* Status: **Awaiting review**
+* Impacted Platforms: [ATF]
+
+## Introduction
+
+This proposal is about modification of Automated Test Framework (ATF) for providing ability to test SDL on remote workstation usinig automated test scripts.   
+
+## Motivation
+
+Automated testing of SDL on the real OEM hardware would significantly reduce the amount of efforts at integration stage. 
+Also, it can ensure that during integration the business logic SDL will be ported successfully and will not be corrupted. 
+
+On various OEM platforms communication between HMI and SDL is performed through different transport, that may not be retrieved from the target (i.e. mqueue). So for ATF it's not possible to connect to SDL from HMI side and emulate HMI behavior.  
```

Comment:
update:
transport to tranports

Reply:
@jordynmackool fixed in 8598a5ec799ab47e187c4d50ea893000afb44130